### DISCUSSION TITLE
I have a question about clean_prefix

### DISCUSSION THREAD

**Discussion by JDJGInc** [Rapptz/discord.py#7360] [General] [answered]
so I use the master branch without an 3rd party extenstions and the prefix te*
can't be cleaned with ctx.clean_prefix, I am wondering why it still is te* either way.
There's no way to escape markdown with just the prefix, so I am guessing it doesn't clean the prefix of stuff with markdown symbols?
and only like mentions?

**Comment by Vexs** [comment, marked as answer]
I don't know what you're expecting clean prefix to do but all it does is clean mentions from the prefix.

**Reply by JDJGInc** [reply]
Okay, this makes pretty sense, thank you :D
so doing escape_markdown on prefixes would do better then right?
